src/ngx_http_modsecurity_rewrite.c

```diff
@@ -215,6 +221,5 @@ ngx_http_modsecurity_rewrite_handler(ngx_http_request_t *r)
         }
     }
 
-
```

Comment:
don't remove space

Reply:
I wonder what reason is to keep doubled whitespace before return action?

Reply:
keep minor change